bentoml/adapters/base_input.py

```diff
@@ -6,17 +6,17 @@
 
 # http://www.apache.org/licenses/LICENSE-2.0
 
+import argparse
+import contextlib
+import functools
 # Unless required by applicable law or agreed to in writing, software
```

Comment:
?

Reply:
I've got no idea what happened there, I couldn't reproduce it with either `dev/format` nor PyCharm reformatting.